source/linux/Foundational_Components/U-Boot/UG-General-Info.rst

```diff
@@ -729,6 +729,12 @@ Build U-Boot
          * tiboot3-j722s-hs-fs-evm.bin from <output directory>/r5
          * tispl.bin, u-boot.img from <output directory>/a53
 
+.. ifconfig:: CONFIG_part_variant in ('AM65X', 'J721E', 'J7200', 'AM64X', 'AM62X', 'AM62AX', 'AM62PX', 'J721S2', 'J784S4','J742S2', 'J722S')
+
+   .. warning::
+
+	Rename "tiboot3-<board>-evm.bin" to "tiboot3.bin" in order for the device to load this binary from the SD card boot partition
```

Comment:
Use `:file:` role and correct indentation.

Reply:
Checkout https://github.com/TexasInstruments/processor-sdk-doc/blob/master/CONTRIBUTING.md#indentation-and-whitespace for indentation